static/js/util/util.js

```diff
@@ -191,6 +193,42 @@ export function validateProfile(profile, requiredFields, messages) {
 }
 
 /* eslint-enable camelcase */
+/*
+check that the profile is complete. we make the assumption that a
+complete profile consists of:
+  - a valid personal tab
+  - an entry for 'currently employed', and a work history entry if
+    `currently employed == 'yes'`
+*/
+export function profileComplete(profile) {
```

Comment:
can do the rename, I'm not sure about returning `false` in the case where there are no errors.

what if I changed the return values to look like `[bool, {...}]`, so if we get all the way to the end the return value is `[true, {}]`, and if we have an error it looks like `[false, { url: "profile/", ... } ]`?